src/main/java/com/library/model/entity/CustomEntity.java

```diff
@@ -0,0 +1,20 @@
+package com.library.model.entity;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@MappedSuperclass
+public class CustomEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    protected Long id;
```

Comment:
Dont know how it work exectly. Please check ID in different tables.
If in some table first ID not equal to 1, it is not good.

Reply:
Ok, I will check this moment